.github/workflows/drivers.yml

```diff
@@ -569,7 +569,7 @@ jobs:
   be-tests-redshift-ee:
     needs: files-changed
     if: github.event.pull_request.draft == false && needs.files-changed.outputs.backend_all == 'true'
-    runs-on: buildjet-2vcpu-ubuntu-2004
+    runs-on: ubuntu-20.04
```

Comment:
I changed this when trying to debug some flaky tests earlier today but it wasn't actually needed so I'm changing it back here